easybuild/easyconfigs/p/PyTorch/PyTorch-0.3.1-intel-2018a-Python-3.6.4.eb

```diff
@@ -0,0 +1,100 @@
+easyblock = 'PythonPackage'
+
+name = 'PyTorch'
+version = '0.3.1'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'http://pytorch.org/'
+description = """Tensors and Dynamic neural networks in Python with strong GPU acceleration.
+PyTorch is a deep learning framework that puts Python first."""
+
+toolchain = {'name': 'intel', 'version': '2018a'}
+
+pytorchdir = 'pytorch-%(version)s'
+extract_cmd_pattern = 'tar -C %s/%s --strip-components=1 -xf %%s'
+
+source_urls = [
+    'https://github.com/pytorch/pytorch/archive',
+    "https://github.com/facebookincubator/gloo/archive",
+    "https://github.com/google/googletest/archive",
+    "https://github.com/nanopb/nanopb/archive",
+    "https://github.com/pybind/pybind11/archive",
+    "https://github.com/wjakob/clang-cindex-python3/archive",
+]
+# PyTorch pulls in a bunch of submodules which don't have releases.
+# We download the submodule revisions from their repos.
+sources = [
+    'v%(version)s.tar.gz',  # PyTorch
+    {
+        'download_filename': 'cb002e4eb8d167c2c60fc3bdaae4e1844e0f9353.tar.gz',
```

Comment:
@wpoely86 @smoors @hajgato @zao How was this determined? Latest & great commit in https://github.com/facebookincubator/gloo ?

Same questions below...

Reply:
If I remember correctly they come from the `.gitsubmodules` files.

Reply:
That helps, see https://github.com/pytorch/pytorch/blob/master/.gitmodules, but it doesn't explain the old `googletest` commit...